crates/sargon-uniffi/src/wrapped_radix_engine_toolkit/low_level/manifest_summary/account_deposit/simple_non_fungible_resource_bounds.rs

```diff
@@ -2,16 +2,8 @@ use crate::prelude::*;
 use sargon::SimpleNonFungibleResourceBounds as InternalSimpleNonFungibleResourceBounds;
 
 /// Represents the bounds for a simple non-fungible resource, which can be either exact or not exact.
-#[derive(Clone, PartialEq, Eq, InternalConversion, uniffi::Enum)]
-pub enum SimpleNonFungibleResourceBounds {
-    Exact {
-        amount: Decimal,
-        certain_ids: Vec<NonFungibleLocalId>,
-    },
-    NotExact {
-        certain_ids: Vec<NonFungibleLocalId>,
-        lower_bound: LowerBound,
-        upper_bound: UpperBound,
-        allowed_ids: AllowedIds,
-    },
+#[derive(Clone, PartialEq, Eq, InternalConversion, uniffi::Record)]
+pub struct SimpleNonFungibleResourceBounds {
+    pub certain_ids: Vec<NonFungibleLocalId>,
+    pub additional_amount: Option<SimpleCountedResourceBounds>,
```

Comment:
Calculated additional_amount that potentially can be received along with `certain_ids`